pkg/lvm/iolimiter.go

```diff
@@ -115,7 +115,7 @@ func getRatePerGB(vgName string, rateMap map[string]uint64) uint64 {
 		return ptr
 	}
 	for k, v := range rateMap {
-		if strings.Contains(vgName, k) {
+		if strings.HasPrefix(vgName, k) {
```

Comment:
shouldn't we use strings.Compare here?

Reply:
Keeping it Prefix match instead of exact to account for the possibility of using multiple pools on a node on the same underlying storage device type. E.g. configuring N JBOD attached HDDs as volumegroups HDD_JBOD_1 through HDD_JBOD_N. However as far as IOPS limits are concerned, it can be provided for HDD_JBOD

Reply:
in that case we should allow users to provide regx in the iops setting and we should do regx comparison here.

Reply:
can we take that as an enhancement? This PR fixes the bug (in the sense that it is asking for prefix but doing substring match). In a subsequent PR, I can introduce regex match and deprecate the current option

Reply:
sure, we can take this in the next PR.